src/style.css

```diff
@@ -1 +1,83 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: bold;
+}
+
+.form {
+  margin-top: auto;
+}
+
+.form-input {
+  display: block;
+  margin: o;
```

Comment:
There is a typo in the `margin` property value; it should be `0` instead of `o`.